timetracker/entries/urls.py

```diff
@@ -5,4 +5,7 @@
 
 urlpatterns = [
     url(r'^entries/$', views.entries, name='entries'),
+    url(r'^clients/$', views.clients, name='clients'),
+    url(r'^projects/$', views.projects, name='projects'),
+    url(r'^client$', views.client, name='client'),
```

Comment:
Typically you would express this url pattern as:

```
    url(r'^client/(?P<client_id>\d+)/$' ...
```
this way the client id is matched from the URL, and passed as an argument to the view function.